### PR TITLE
Vendor: projects

### PR DESCRIPTION
**Pull request by chadwcarlson**
Separates `projects` dir.

### CODE REVIEW

File: sites/friday/src/projects/region-migration.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+If you have files in a mount, first download them:
+
+```bash
+{{% vendor/cli %}} mount:download
+```
+
+Then upload them to your new project:
+
+```bash
+{{% vendor/cli %}} mount:upload
```
Comment (flovntp) [inline sites/friday/src/projects/region-migration.md:89]: `mount:upload` alone is not working so maybe display the doc page snippet?
```
{{% vendor/cli %}} mount:upload --mount private --source ./private
```

File: sites/friday/src/projects/region-migration.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+Your new project then mirrors the configured repository automatically.
+
+{{< /codetabs >}}
+
+## 4. Copy files
+
+If you have files in a mount, first download them:
+
+```bash
+{{% vendor/cli %}} mount:download
```
Comment (flovntp) [inline sites/friday/src/projects/region-migration.md:83]: `mount:download` alone is not working, so maybe display the doc page snippet ?
```
{{% vendor/cli %}} mount:download --mount private --target ./private
```

File: sites/friday/src/projects/region-migration.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+Verify that the new site is working as desired before continuing.
+You can leave the two projects running for as long as you need.
+After you have finished all your testing, sync all your data (code, files, database) for the last time.
+
+## 8. Switch to the new site
+
+Now that you know the new project works, switch public traffic to that site:
+
+1. Make sure your new project has the right plan size.
+2. If possible, put your site into read-only mode or maintenance mode.
```
Comment (flovntp) [inline sites/friday/src/projects/region-migration.md:128]: as for custom SSL certificates, add `(Optional)` at the beginning?

File: sites/friday/src/projects/change-project-timezone.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- [App runtime](../create-apps/timezone.md).
+- [Cron jobs](../create-apps/app-reference.md#crons).
+- [System logs](../increase-observability/logs/_index.md). UTC is the default timezone for all logs.
+
+To change the timezone for a project, follow these steps:
+
+1. Open the project where you want to change the timezone.
+2. Click {{< icon settings >}} **Settings**.
+3. Select the timezone from the list.
+4. Click **Save**.
```
Comment (flovntp) [inline sites/friday/src/projects/change-project-timezone.md:19]: maybe we need to add a link (to this page) into the `security/backups.html` page, and add a note on that backup page to let the reader know that the schedule of backups is based on the project timezone?

File: sites/friday/src/projects/region-migration.md (hunk 1 above)
Comment (gilzow) [reply sites/friday/src/projects/region-migration.md:89]: @flovntp I'm not following. `mount:upload` and `mount:download` will interactively prompt you for which mount, as well as the target directory (`source`).

**Review by gilzow — DISMISSED** [review]
VRT tests pass

File: sites/friday/src/projects/region-migration.md (hunk 3 above)
Comment (gilzow) [reply sites/friday/src/projects/region-migration.md:128]: It does, unless I'm not understanding? 
`4. (Optional) Add any custom SSL certificates you have.`

**Review by gilzow — APPROVED** [review]

**Pull request merged**